pyschism/forcing/hycom/hycom2schism.py

```diff
@@ -727,104 +727,208 @@ def fetch_data(self, outdir: Union[str, os.PathLike], hgrid, vgrid, start_date,
 
 class DownloadHycom:
 
-    def __init__(self, hgrid):
-
-        xmin, xmax = hgrid.coords[:, 0].min(), hgrid.coords[:, 0].max()
-        ymin, ymax = hgrid.coords[:, 1].min(), hgrid.coords[:, 1].max()
+    def __init__(self, hgrid=None):
+        if hgrid is not None:
+            xmin, xmax = hgrid.coords[:, 0].min(), hgrid.coords[:, 0].max()
+            ymin, ymax = hgrid.coords[:, 1].min(), hgrid.coords[:, 1].max()
+        else:
+            xmin, ymin, xmax, ymax = -180., -90, 180., 90.
         xmin = xmin + 360. if xmin < 0 else xmin
```

Comment:
The indexing error goes away when I remove these lines. Is this a conditionally necessary conversion?